docker-compose.override.test.yml

```diff
@@ -0,0 +1,33 @@
+version: '3.7'
+
+services:
+
+  app:
+    environment:
+      # run flush, migrate and collectstatic in entrypoint
+      DEBUG: 1
+      # we must use the same db for test and runserver
+      SQL_DATABASE: test_${POSTGRES_DB}
+      SQL_DATABASE_TEST: test_${POSTGRES_DB}
+    command: python3 manage.py runserver 0.0.0.0:8000
+
+  worker_wrapper:
+    environment:
+      # we must use the same db for test and runserver
+      SQL_DATABASE: test_${POSTGRES_DB}
+      SQL_DATABASE_TEST: test_${POSTGRES_DB}
+
+  db:
+    environment:
+      POSTGRES_DB: test_${POSTGRES_DB}
+
+networks:
+  default:
+    # Use a custom driver
+    name: ${QFIELDCLOUD_DEFAULT_NETWORK}
```

Comment:
Is that needed ? If so how come it's only needed for tests ?

Reply:
Yes, it is. I use it to have multiple compose qfieldclouds on the same machine. Since the `COMPOSE_PROJECT_NAME` is also different, I need to update the network name too.

Reply:
Configured only for `test`, as the default is properly set on the normal setup.